exonum-java-binding-cryptocurrency-demo/frontend/server.js

```diff
@@ -1,44 +1,36 @@
-var express = require('express');
-var bodyParser = require('body-parser');
+var express = require('express')
+var proxy = require('http-proxy-middleware')
 
 // Initialize application
-var app = express();
+var app = express()
 
 // Get app params
-var argv = require('yargs-parser')(process.argv.slice(2));
-var port = argv.port;
-var explorerRoot = argv.explorerRoot;
-var apiRoot = argv.apiRoot;
+var argv = require('yargs-parser')(process.argv.slice(2))
+var port = argv.port
+var explorerRoot = argv.explorerRoot
+var apiRoot = argv.apiRoot
 
 if (typeof port === 'undefined') {
-  throw new Error('--port parameter is not set.');
+  throw new Error('--port parameter is not set.')
```

Comment:
maybe we need to add `jslint` to the project to avoid such things?

Reply:
Could you change var to let or const?

Reply:
@bullet-tooth we already started a process of moving all project to same linting rules

Reply:
@bullet-tooth sure